OneCommunityGlobal/HighestGoodNetworkApp
# Shefali unit test pie chart

# Description
Unit test for src/components/common/PieChart/PieChart.jsx


## Main changes explained:
Added test cases to check for the following
-  applies dark mode class correctly
- renders the SVG pie chart
- generates the correct number of unique colors
- -removes SVG on unmount
- 
## How to test:
1. check into current branch
2. do `npm install`
3. do `npm test PieChart.test.js`

##Screenshot after running:
![Screen Shot 2024-09-13 at 12 21 35 PM](https://github.com/user-attachments/assets/f9ab5479-ac21-4779-939d-11b255dc065b)
